frontend/packages/pipelines-plugin/src/components/pipelines/detail-page-tabs/pipeline-details/PipelineVisualization.tsx

```diff
@@ -32,7 +31,12 @@ const PipelineVisualization: React.FC<PipelineTopologyVisualizationProps> = ({
         )}
       />
     );
-  } else if (nodes.length === 0 && edges.length === 0) {
+    return <div className="odc-pipeline-visualization">{content}</div>;
```

Comment:
Remove this duplicate `return` statement. Since you already assign the `content`, add an `} else {` statement after this to block the rest of the conditions and continue to use the single `return` statement.